### PR TITLE
[Update Rytec-UK-Ireland.xml] Fix typo

### PR DESCRIPTION
**Pull request by I-Cat**
Fix typo

### CODE REVIEW

**Comment by I-Cat** [comment]
Thx

**Comment by I-Cat** [comment]
Odd. I wonder why fixing a typo failed

File: pvr.iptvsimple/resources/data/genres/genreTextMappings/Rytec-UK-Ireland.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!--
-Note: the first 4 bits is genre and last is sub genre
+Note: the first 4-bits is genre and last is sub genre
```
Comment (phunkyfish) [inline pvr.iptvsimple/resources/data/genres/genreTextMappings/Rytec-UK-Ireland.xml]: This is not a typo, the dash should not be added.

File: pvr.iptvsimple/resources/data/genres/genreTextMappings/Rytec-UK-Ireland.xml [hunk 2]
```diff
@@ -9,7 +9,7 @@ Mapping Rytec Text Genres:
  - If a mapping cannot be found the text will be used but there will be no colouring (Genre Description only)
  - All text will be matched in lowercase
 
-If you are creating yoru own text mappings make a copy of this file in the same directory so it's not overwritten and start from there.
+If you are creating you'r own text mappings make a copy of this file in the same directory so it's not overwritten and start from there.
```
Comment (phunkyfish) [inline pvr.iptvsimple/resources/data/genres/genreTextMappings/Rytec-UK-Ireland.xml]: `your`

**Review by phunkyfish — CHANGES_REQUESTED** [review]

File: pvr.iptvsimple/resources/data/genres/genreTextMappings/Rytec-UK-Ireland.xml (hunk 1 above)
Comment (I-Cat) [reply pvr.iptvsimple/resources/data/genres/genreTextMappings/Rytec-UK-Ireland.xml]: Okay Thank You

File: pvr.iptvsimple/resources/data/genres/genreTextMappings/Rytec-UK-Ireland.xml (hunk 2 above)
Comment (I-Cat) [reply pvr.iptvsimple/resources/data/genres/genreTextMappings/Rytec-UK-Ireland.xml]: Even better

**Review by I-Cat — COMMENTED** [review]
fix

**Review by I-Cat — COMMENTED** [review]
[Viewed]

**Comment by I-Cat** [comment]
ok So I need to close this one.

**Pull request closed without merging**